src/plugins/LADTable/components/LadTableSet.vue

```diff
@@ -66,6 +68,24 @@ export default {
             compositions: []
         }
     },
+    computed: {
+        hasUnits() {
+            for(let telemetryObject in this.secondaryTelemetryObjects) {
+                if(this.secondaryTelemetryObjects[telemetryObject]) {
+                    let objects = this.secondaryTelemetryObjects[telemetryObject];
```

Comment:
isn't `telemetryObject` already an object in `this.secondaryTelemetryObjects`?

Reply:
whenever we use "for in" there's a linting rule "guard-for-in" which makes you check if the property exists... I'm not too keen on that one.

Reply:
Reworked this to make it more clear and clean.